lib/modules/manager/maven-wrapper/extract.spec.ts

```diff
@@ -2,11 +2,19 @@ import { extractPackageFile } from '.';
 
 const onlyWrapperProperties =
   'wrapperUrl=https://repo.maven.apache.org/maven2/io/takari/maven-wrapper/0.5.6/maven-wrapper-0.5.6.jar';
+
+const onlyWrapperVersionProperties =
+  'wrapperVersion=https://internal.artifactory.acme.org/artifactory/maven-bol/org/apache/maven/wrapper/maven-wrapper/3.3.0/maven-wrapper-3.3.0.jar';
```

Comment:
The value of the `wrapperVersion` property should be just a version number, so for example: `wrapperVersion=3.3.0`.

Reply:
The trickiest case is `maven-wrapper-3.3.0-only-script`, because the has `wrapperUrl` nor `wrapperVersion`:

```
distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.6/apache-maven-3.9.6-bin.zip
```

I believe the Maven Wrapper version for `maven-wrapper-3.3.0-only-script` can only be detected by looking for the string `Apache Maven Wrapper startup batch script, version 3.3.0` in `mvnw` (or `mvnw.cmd` if `mvnw` is not present).

Reply:
I'm currently trying to find out if Maven Wrapper provides a way to update Maven Wrapper using the existing installation type, without having to explicitly specify (and thus determine) it. If so, Renovate should update Maven Wrapper like that.

If upgrading using the existing installation type can only be done by specifying the type in the update command (`-Dtype=${type}`), then I plan to request a property for that too (e.g. `wrapperType`). But for older versions that don't explicitly communicate their installation type, Renovate might still need to detect the installation type. I suggest keeping this installation type business out of scope for this current PR. That could be added later if needed.

Reply:
How does it figure out on which url it should get the maven-wrapper ? Or is that now all covered in mvn ?